cs/ccxt/base/Throttler.cs

```diff
@@ -3,32 +3,48 @@
 namespace ccxt;
 
 using dict = Dictionary<string, object>;
-public class Throttler
+
+public class ThrottlerConfig
 {
+    public double RefillRate { get; set; } = 1.0;
```

Comment:
we might need to add some comments or something to tell which options are part of the leaky bucket and which belong to the rolling window